<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stbl_widget</class>
 <widget class="QWidget" name="stbl_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Structure Blending</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="Obj2Graph">
      <attribute name="title">
       <string>Object To Graph</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Voxel resolution:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="VoxelResolution"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="LoadSingle">
         <property name="text">
          <string>Load Single Segmented Object</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SS">
      <attribute name="title">
       <string>Split Shapes</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="SplitSingle">
         <property name="text">
          <string>Load single shape and split</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="SplitList">
         <property name="text">
          <string>Load a List to split</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="Filtering">
         <property name="text">
          <string>Filter files</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Project">
      <attribute name="title">
       <string>Project Image</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="LoadSigleMesh">
         <property name="text">
          <string>Load Single mesh to test</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="cameraIndex"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="GenSingleImage">
         <property name="text">
          <string>Generate image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cameraPath">
         <property name="text">
          <string>UtilityLib\cameras</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Camera File Path: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="LoadAList">
         <property name="text">
          <string>Batch processing</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Corr">
      <attribute name="title">
       <string>Correspondence</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="1">
        <widget class="QListView" name="TargetPartSet"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="LoadInbet">
         <property name="text">
          <string>Load inbetween templates</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="HasInbetInfo">
         <property name="text">
          <string>Has Inbetween Information</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="GenPartSet">
         <property name="text">
          <string>Generate candidates part set</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="HasPartInfo">
         <property name="text">
          <string>Has Part Information</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="LoadPair">
         <property name="text">
          <string>Load shape pair</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="FindInbet">
         <property name="text">
          <string>Find inbetween templates</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QListView" name="SourcrPartSet"/>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="FindCorr">
         <property name="text">
          <string>Find Correspondence</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QListView" name="CorrSet"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="structureblending_mode.qrc"/>
 </resources>
 <connections/>
</ui>
